CMakeLists.txt

```diff
@@ -3,15 +3,18 @@
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 # cmake-format: on
 
-cmake_minimum_required(VERSION 3.25...3.31)
+cmake_minimum_required(VERSION 3.30...3.31)
 
-project(beman_execution26 VERSION 0.0.1 LANGUAGES CXX)
+project(beman_execution26 VERSION 0.1.0 LANGUAGES CXX)
 
 if(${CMAKE_SOURCE_DIR} STREQUAL ${CMAKE_BINARY_DIR})
   message(FATAL_ERROR "In-source builds are not allowed!")
 endif()
 
-set(CMAKE_CXX_SCAN_FOR_MODULES ON)
+# Note that scanning is only performed if C++20 or higher is enabled for the target. Scanning for modules in the target's
+# sources belonging to file sets of type CXX_MODULES is always performed.
+# not needed: set(CMAKE_CXX_SCAN_FOR_MODULES ON)
+
```

Comment:
I had problems without this definition. Adding it resulted in things building on MacOS. Probably this part of the code can be removed.

Reply:
yes